src/seismometer/data/binary_performance.py

```diff
@@ -0,0 +1,125 @@
+from enum import Enum
+from typing import List, Optional
+
+import numpy as np
+from numpy.typing import ArrayLike
+from sklearn.metrics import auc
+
+from . import calculate_bin_stats
+
+GENERATED_COLUMNS = {
+    "positives": "Positives",
+    "prevalence": "Prevalence",
+    "auroc": "AUROC",
+    "auprc": "AUPRC",
+    "accuracy": "Accuracy",
+    "ppv": "PPV",
+    "sensitivity": "Sensitivity",
+    "specificity": "Specificity",
+    "flagged": "Flag Rate",
+    "threshold": "Threshold",
+}
+
+
+class Metric(Enum):
+    """
+    Enumeration for available values for metric parameter in PerformanceMetrics class.
+    """
+
+    Sensitivity = "sensitivity"
+    Specificity = "specificity"
+    Flagged = "flagged"
+    Threshold = "threshold"
+
```

Comment:
Should we move this into performance, or update performance to pull these values then we can not have multiple places defining key metric names?

Also can we rename Flagged to FlagRate?

Reply:
Changed the class name to MonotonicMetric and moved it to performance.py.